packages/vite-config/package.json

```diff
@@ -34,15 +34,13 @@
     }
   },
   "devDependencies": {
-    "@esbuild-plugins/node-globals-polyfill": "^0.2.3",
-    "@esbuild-plugins/node-modules-polyfill": "^0.2.2",
     "@svgr/plugin-jsx": "^8.0.1",
-    "@vitejs/plugin-react": "^4.0.3",
-    "rollup-plugin-polyfill-node": "^0.12.0",
+    "@vitejs/plugin-react": "^4.2.1",
     "svgo": "^3.0.2",
-    "typescript": "5.1.6",
-    "vite": "^4.5.2",
-    "vite-plugin-svgr": "^3.2.0",
+    "typescript": "5.4.2",
+    "vite": "^5.1.6",
+    "vite-plugin-node-polyfills": "0.17.0",
```

Comment:
This plugin allows us to replace both the esbuild and rollup Node polyfill plugins.

Using v0.17 because there is a bug in later versions (currently v0.21)